### DISCUSSION TITLE
Unable to login to hue with saml integration in 4.11. was able to login in 4.7.1

### DISCUSSION THREAD

**Discussion by rameshkumarchikoti** [cloudera/hue#3187] [Questions]
In 4.11 hue version when I integrate saml to hue when I login I am seeing below errors in hue pod
`ERROR    Failed to create home directory for user hue: HTTPConnectionPool(host='localhost', port=50070): Max retries exceeded with url: /webhdfs/v1?user.name=hue&doas=hue&op=GETHOMEDIRECTORY (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbb3cfd3340>: Failed to establish a new connection: [Errno 111] Connection refused'))
Traceback (most recent call last):`

In 4.7.1 hue version we were able to integrate saml and able to login.

**Comment by Harshg999** [comment]
Hi @rameshkumarchikoti, the error is not related to SAML integration in Hue. 
Its trying to get or create the home directory for user in HDFS but not able to connect to HDFS.
